src/__tests__/components/NavBar/EditorDesignName.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { render, screen, fireEvent } from "@testing-library/react";
```

Comment:
You can use userEvent, check this [link](https://testing-library.com/docs/user-event/intro/#difference-to-fireevent)

Reply:
Thank you for the suggestion. Let me learn it and use it in the next card.

Reply:
No need to use it in the next card, you can use it in this one